app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -47,8 +46,8 @@ class ProfileFragment : Fragment() {
         findNavController(rootView).navigate(R.id.loginFragment, bundle, getAnimFade())
     }
 
-    private fun redirectToMain() {
-        startActivity(Intent(activity, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP))
+    private fun redirectToEvents() {
```

Comment:
`redirectToEventsFragment`

Reply:
renamed same in OrderCompletedFragment also